catalog/plugin/catalog-plugin-duplication/src/main/java/ddf/catalog/metacard/duplication/DuplicationValidator.java

```diff
@@ -36,6 +36,7 @@
 import ddf.catalog.CatalogFramework;
 import ddf.catalog.data.Attribute;
 import ddf.catalog.data.Metacard;
+import ddf.catalog.data.types.Validation;
```

Comment:
fixed, the code formatter didn't catch it because it was being used in a javadoc